### PR TITLE
Scrutinizer fixes and some clean up

### PR DESCRIPTION
**Pull request by dpslwk**

### CODE REVIEW

**Comment by dpslwk** [comment]
travis fix is in #215

File: app/HMS/Entities/Invite.php [hunk 1]
```diff
@@ -64,7 +64,7 @@ public function getEmail(): string
 
     /**
      * Get the token value.
-     * @return ?string
+     * @retrun null|string
```
Comment (geeksareforlife) [inline app/HMS/Entities/Invite.php]: return

File: app/HMS/Entities/Email.php [hunk 2]
```diff
@@ -176,7 +176,7 @@ public function getUsers()
     /**
      * Gets the value of role.
      *
-     * @return ?Role
+     * @retrun null|Role
```
Comment (geeksareforlife) [inline app/HMS/Entities/Email.php]: return

File: app/HMS/Entities/Profile.php [hunk 3]
```diff
@@ -77,15 +77,15 @@ public function setUser(User $user): Profile
     }
 
     /**
-     * @return ?Carbon
+     * @retrun null|Carbon
```
Comment (geeksareforlife) [inline app/HMS/Entities/Profile.php]: return

File: app/HMS/Entities/Profile.php [hunk 4]
```diff
@@ -96,15 +96,15 @@ public function setJoinDate(?Carbon $joinDate): Profile
     }
 
     /**
-     * @return ?string
+     * @retrun null|string
```
Comment (geeksareforlife) [inline app/HMS/Entities/Profile.php]: return

File: app/HMS/Entities/Profile.php [hunk 5]
```diff
@@ -134,15 +134,15 @@ public function setCreditLimit(int $creditLimit): Profile
...
```
Comment (geeksareforlife) [inline app/HMS/Entities/Profile.php]: return

File: app/HMS/Entities/Profile.php [hunk 6]
```diff
@@ -267,7 +267,7 @@ public function setContactNumber(?string $contactNumber): Profile
     }
 
     /**
-     * @return ?Carbon
+     * @retrun null|Carbon
```
Comment (geeksareforlife) [inline app/HMS/Entities/Profile.php]: return

File: app/HMS/Entities/RoleUpdate.php [hunk 7]
```diff
@@ -62,7 +62,7 @@ public function getUser(): User
     /**
      * Gets the value of roleAdded.
      *
-     * @return ?Role
+     * @retrun null|Role
```
Comment (geeksareforlife) [inline app/HMS/Entities/RoleUpdate.php]: return

File: app/HMS/Entities/RoleUpdate.php [hunk 8]
```diff
@@ -72,7 +72,7 @@ public function getRoleAdded(): ?Role
     /**
      * Gets the value of roleRemoved.
      *
-     * @return ?Role
+     * @retrun null|Role
```
Comment (geeksareforlife) [inline app/HMS/Entities/RoleUpdate.php]: return

File: app/HMS/Entities/User.php [hunk 9]
```diff
@@ -218,15 +218,15 @@ public function getPermissions()
     }
 
     /**
-     * @return ?Profile The users profile
+     * @retrun null|Profile The users profile
```
Comment (geeksareforlife) [inline app/HMS/Entities/User.php]: return

File: app/HMS/Entities/User.php [hunk 10]
```diff
@@ -237,15 +237,15 @@ public function setProfile(?Profile $profile): User
     }
 
     /**
-     * @return ?Account The users account
+     * @retrun null|Account The users account
```
Comment (geeksareforlife) [inline app/HMS/Entities/User.php]: return

**Pull request merged**